internal/config/config.go

```diff
@@ -29,6 +29,11 @@ type App struct {
 type Auth struct {
 	Token          string
 	TokenHeaderKey string
+	Router         struct {
+		ValidationURL   string
+		ValidationToken string
+		CachedInterval  string
```

Comment:
rename to CacheTTL

Reply:
suffix time unit 
eg: CacheTTLSeconds or CacheTTLMinutes

Reply:
ok , Making it CacheTTLMinutes